.github/CONTRIBUTING.md

```diff
@@ -92,10 +92,22 @@ Perfect, you are setup and ready to go.
 Have a look at [help wanted issues](https://github.com/robstoll/atrium/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
 where [good first issues](https://github.com/robstoll/atrium/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
 are easier to start with.
-Please write a comment such like `I am working on this` into the issue, 
+Please write a comment such as `I am working on this` in the issue,
 this way we can assign the task to you (so that others know there is already someone working on the issue)
 and it gives us the chance to have a look at the description again and revise if necessary.
 
+When working on an issue, we recommend you do the following on git:
+- create a new branch for the issue using `git checkout -b <branch-name>` (preferrably, the branch name
+  should be descriptive of the issue or the change being made, e.g `#200-add-feature`.) Working
```

Comment:
Maybe `#200-add-feature` is a bad example because somebody could take it literally. We could change it to something like `#108-path-exists`.